daemon/config.go

```diff
@@ -38,6 +38,7 @@ type Config struct {
 	BpfDir                string                  // BPF template files directory
 	LibDir                string                  // Cilium library files directory
 	RunDir                string                  // Cilium runtime directory
+	LXCMapMU              sync.Mutex              // LXCMapMU is the mutex to access the LXCMap FIXME move this to the LXCMap struct?
```

Comment:
Moved to LXCMap struct. Created an issue to have a mutex in the generic BPF map abstraction.